addons/SmoothScroll/SmoothScrollContainer.gd

```diff
@@ -630,6 +630,21 @@ func scroll_to_left(duration:float=0.5) -> void:
 func scroll_to_right(duration:float=0.5) -> void:
 	scroll_x_to(self.size.x - content_node.size.x, duration)
 
+func scroll_to_fauna(duration:float=0.5) -> void:
+	if(!PlayerManager.is_player_initial_position_set):
+		return
+	
+	var room_distance: float = 66
+	var scroll_to_pos:float = -room_distance
+	var rooms_between_fauna_and_top = MapManager.map_width_array.size() - 8
```

Comment:
why 8 ?

Reply:
It's that many slots which are shown when you are scrolled to the top, I'll put it in a variable

Reply:
why are you using the map width array instead of the map height then ?
and yes, naming numbers is better

Reply:
I couldn't find the map height

Reply:
ah yes sorry, the size of the `map_width_array` is indeed the height of the map, since it's an array of the width of all the floors